ui/perfherder/tests/TestsTableControls.jsx

```diff
@@ -79,6 +79,7 @@ export default class TestsTableControls extends React.Component {
 }
 
 TestsTableControls.propTypes = {
+  validated: PropTypes.shape({}),
```

Comment:
I have a question about the use of `validated` here.

I was trying to check for the data type in the react dev tools but noticed `validated` is not passes as a prop to the `TestsTableControls` component when used in `TestsView` component (See below)

https://github.com/mozilla/treeherder/blob/d6ba99312cb954bf07db3c0f6f769f72ca7cdfe0/ui/perfherder/tests/TestsView.jsx#L145-L150

This prop seems to be redundant, is there a reason why it is been used?

Reply:
@ionutgoldan @airimovici do you have an answer for this?

Reply:
@ionutgoldan @airimovici : re-ping on this question to you.  :)

Reply:
I really don't know how that property ended up there. @bolah2009 feel free to remove it, as it's confusing.

Reply:
Okay, thanks @ionutgoldan